zkvm/src/types.rs

```diff
@@ -264,13 +285,20 @@ impl Data {
     }
 
     /// Encodes blinded Data values.
```

Comment:
Why is it "blinded data values"? BTW, this length prefix is only used in the output serialization, why don't we just use `data.to_bytes` there and add a length prefix where we encode the output? This method in its current shape can be confused with `to_bytes`.

Reply:
I think we use "blinded data values" here to refer to the fact that we only encode the blinded/opaque versions of data? open to a more descriptive comment though!

Reply:
To avoid confusion, i'd use "blinded" only to places where we blind Pedersen commitments. For opaque "serialized" representations we could use the "opaque" term.